### ISSUE TITLE
Morning session peer-to-peer code review

### ISSUE THREAD

**Issue by Besufekad-HAZ** [Koshao-sys/Personal-Portfolio-Site#13]
# Using an Alt text can be important for SEO, Website Accessibility, and Text placeholder when images are slow to load or fail to load.
https://github.com/Koshao-sys/Personal-Portfolio-Site/blob/7bca8c00eefeee4a9ca23621ad4365d708b97a4b/index.html#L41-L45
# Leaving Unnecessary comment lines can affect the cleanness of your code so please try to remove them
https://github.com/Koshao-sys/Personal-Portfolio-Site/blob/7bca8c00eefeee4a9ca23621ad4365d708b97a4b/index.html#L141
